frontend/src/utils/formatPath.ts

```diff
@@ -23,7 +23,12 @@ export const parseBasePath = (value = basePathMetaTagContent()): string => {
 
 // Join paths with a leading separator and without a trailing separator.
 const joinPaths = (...paths: string[]): string => {
-    return ['', ...paths]
+    const filteredPaths = paths.filter(path => {
+        return !paths.some(i => i !== path && i.includes(path));
```

Comment:
Could we be more explicit here, I find `i` to be a bit confusing in this context. `currentPath` ?